README.md

```diff
@@ -78,6 +78,50 @@ Environment: AWS EC2 p2.medium and running the operations with Fuseki triple sto
 * Calculating the diff for two versions of a large NAL (used Corporate Bodies) ~ 58s
 * Generating the diff report for two versions of a large NAL (used Corporate Bodies)  ~  12 min
 
+# Change type inventory
+
+This section provides a change type inventory along with the patterns captured by each change type. We model the change as state transition operator between old (on the left) and teh new (on the right). The transition operator is denoted by the arrow symbol (-->). On each sides of the transition operator, we use a compact notation following SPARQL triple patterns. 
+
+We use a set of conventions for each variable in the triple pattern, ascribing meaning to each of them and a few additional notations. These conventions are presented in teh table below.
+
+| Notation                   | Meaning                                                                                                                                                    | Example                 |
+|----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
+| triple pattern < _s p o_ >   | each item in the triple represents a SPARQL variable or an URI. For brevity we omit the question mark prefix (?) otherwise the SPARQL reading shall apply. | i p v                   |
+| arrow (_ --> _)              | state transition operator (from one version to the next)                                                                                                   | i1 p o  -->  i2 p o     |
+| _i_ - in the triple pattern  | the instance subject (assuming class instantiation)                                                                                                        | i p v                   |
+| _p_ - in the triple pattern  | the main predicate                                                                                                                                         | i p v                   |
```

Comment:
I recommend using the `code` text style for the **Notation** and **Example** columns as I think it should improve readability.
![table](https://i.imgur.com/2DVArSN.png)